insights/client/connection.py

```diff
@@ -77,6 +77,25 @@ def _api_request_failed(exception, message='The Insights API could not be reache
         logger.error(message)
 
 
+def _pkg_name_version(name):
+    try:
+        import importlib.metadata
+    except ImportError:
+        # Python < 3.8
+        import pkg_resources
+        pkg = pkg_resources.working_set.find(pkg_resources.Requirement.parse(name))
+        if pkg is not None:
+            return pkg.project_name, pkg.version
+    else:
+        try:
+            pkg = importlib.metadata.distribution(name)
+            return pkg.name, pkg.version
+        except ModuleNotFoundError:
+            pass
+    return None, None
+
+
+
```

Comment:
Probably too many empty lines, this would not be accepted by `flake8`/`black` pipeline. 2 empty lines is enough